scio-tensorflow/src/main/protobuf/schema.proto

```diff
@@ -0,0 +1,335 @@
+// Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =============================================================================
+
+
+syntax = "proto2";
```

Comment:
do we know if they package this schemas? I'm not against the copy ... I'm just wondering how we keep update with the schema and how do we check we are not up2date.

Reply:
+1, it's unlikely that they change this schema but if they do :\